doc/source/ray-air/tuner.rst

```diff
@@ -0,0 +1,216 @@
+.. _air-tuner:
+
+Configuring Hyperparameter Tuning
+=================================
+
+The Ray AIR :class:`Tuner <ray.tune.Tuner>` is the recommended way to tune hyperparameters in Ray AIR.
+
+.. image:: images/tuner.svg
+    :align: center
+
+`Tuner` provides an interface that works with AIR Trainers to perform distributed
+hyperparameter tuning. `Tuner` is part of Ray Tune and provides
+a variety of state-of-the-art hyperparameter tuning algorithms for optimizing model
+performance.
+
+This page will cover basic usage examples of a Tuner. If you are interested in
```

Comment:
Not a fan of "This page will..."

I would rephrase as: 

"What follows next is basic coverage of what a Tuner is and how you can use it for basic examples.